loadflow/loadflow-validation/src/main/java/com/powsybl/loadflow/validation/ValidationTool.java

```diff
@@ -139,7 +144,7 @@ public Command getCommand() {
     public void run(CommandLine line, ToolRunningContext context) throws Exception {
         Path caseFile = Paths.get(line.getOptionValue(CASE_FILE));
         Path outputFolder = Paths.get(line.getOptionValue(OUTPUT_FOLDER));
-        if (!Files.exists(outputFolder)) {
+        if (!outputFolder.toFile().exists()) {
```

Comment:
I’m sure you have to follow sonar advice for this smell, in particular if the Path has been created from a file system implementation that cannot give you a File (JimFS, remote file system...)

Reply:
So your suggestion is, despite the sonarqube major issue, to use Files.exists, in order to possibly cover, and run the itool command in, the different file system implementations?

Reply:
Exactly. We are not fully inline with this sonar suggestion, so while we don't find the best thing to do, use Files.exists(Path)